module/utils.mjs

```diff
@@ -1061,18 +1070,38 @@ export function getHumanReadableAttributeLabel(attr, { actor }={}) {
   }
 
   // Check if the attribute is already in cache.
-  let label = _attributeLabelCache.get(attr);
+  let label = item ? null : _attributeLabelCache.actor.get(attr);
   if ( label ) return label;
+  let name;
+  let type = "actor";
 
-  // Derived fields.
-  if ( attr === "attributes.init.total" ) label = "DND5E.InitiativeBonus";
-  else if ( (attr === "attributes.ac.value") || (attr === "attributes.ac.flat") ) label = "DND5E.ArmorClass";
-  else if ( attr === "attributes.spelldc" ) label = "DND5E.SpellDC";
+  const getSchemaLabel = (attr, type) => {
+    for ( const model of Object.values(CONFIG[type].dataModels) ) {
+      const field = model.schema.getField(attr);
+      if ( field ) return field.label;
+    }
+  };
 
-  // Abilities.
-  else if ( attr.startsWith("abilities.") ) {
-    const [, key] = attr.split(".");
-    label = game.i18n.format("DND5E.AbilityScoreL", { ability: CONFIG.DND5E.abilities[key].label });
```

Comment:
I tested this change, but it's not correct:

```js
getHumanReadableAttributeLabel("abilities.dex.value", { actor: _token.actor }); // "Ability Score"
```

Reply:
Yeah, I think some lines might have been lost during a rebase or something while working on this, because these lines shouldn't have been removed.